documentation/API_documentation/NumberVerification-Readiness-Checklist.md

```diff
@@ -0,0 +1,25 @@
+# API Readiness Checklist
+
+Checklist for number_verification in version b1.0.0-rc1.1
+
+| Nr | API release assets  | alpha | release-candidate |  public-release<br>initial | public-release<br> stable | Status | Comments |
+|----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|
+|  1 | API definition                               |   M   |         M         |    M    |    M   |   Y   | [link](https://github.com/camaraproject/NumberVerification/blob/main/code/API_definitions/number_verification.yaml) |

```

Comment:
Best practice is to use relative URL (which are pointing then to the current branch or tag):

```suggestion
|  1 | API definition                               |   M   |         M         |    M    |    M   |   Y   | [link](/code/API_definitions/number_verification.yaml) |

```